src/app/public/testing/radio-fixture.ts

```diff
@@ -0,0 +1,188 @@
+import {
+  By
+} from '@angular/platform-browser';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  DebugElement
+} from '@angular/core';
+
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+/**
+ * Allows interaction with a SKY UX radio buttons within a radio group.
+ */
+export class SkyRadioFixture {
+
+  private debugEl: DebugElement;
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    private skyTestId: string
+  ) {
+    this.debugEl = SkyAppTestUtility.getDebugElementByTestId(this.fixture, this.skyTestId, 'sky-radio-group');
+  }
+
+  /**
+   * The selected radio button value.
+   */
+  public get value(): any {
+    const selectedRadio = this.debugEl.query(By.css('sky-radio input:checked'));
+    const selectedValue = selectedRadio && selectedRadio.nativeElement.value;
+
+    return selectedValue;
+  }
+
+  /**
+   * The selected radio button label.
+   */
+  public get selectedLabel(): string {
+    const allRadioButtons = this.getAllSkyRadioButtonEls();
+    const selectedRadioInputValue = this.value;
+
+    for (let i = 0; i < allRadioButtons.length; i++) {
+      const skyRadio = allRadioButtons[i];
+      const inputEl = skyRadio.query(By.css('input'));
+
+      if (inputEl.nativeElement && inputEl.nativeElement.value === selectedRadioInputValue) {
+        return this.getRadioButtonLabelText(i);
+      }
+    }
+  }
+
+  /**
+   * Get the value of the radio button at the given index.
+   * @param index The index of the radio button.
+   */
+  public getRadioButtonValue(index: number): any {
```

Comment:
I'd rename this to `getValueByIndex` to make the method name more descriptive.

Also, this method feels a bit odd to me. Is there a current customer requirement to get the value of a given radio button in a group? I feel like setting the radio group to a specific value, and retrieving the value of the radio group would suffice?